### PR TITLE
Added Light Vehicles queue.

### LINKED ISSUE #752: Separate Main Base and Factory queues
Producing units in the Main Base doesn't allow simultaneous production in the factory and vice versa. This is counterintuitive because these are separate buildings.

Suggestion: add an icon to units produced in the Main Base similar to pod weapons to disntinguish them from regular factory units.

image

### PR DESCRIPTION
**Pull request by Dzierzan**
This PR seperates Miners and Builders from the rest of offensive vehicles. This allows building both economy and offsensive vehicle actors. Mind that AI also acquired such ability.

image

Closes #752

### CODE REVIEW

File: mods/hv/chrome/ingame-player.yaml [hunk 1]
```diff
@@ -550,12 +550,27 @@ Container@PLAYER_WIDGETS:
...
+						ProductionTypeButton@LIGHTVEHICLES:
 							Y: 93
 							Width: 28
 							Height: 27
 							VisualHeight: 0
 							Background: sidebar-button
+							TooltipText: Light Vehicles
+							TooltipContainer: TOOLTIP_CONTAINER
+							ProductionGroup: LightVehicles
+							Key: ProductionTypeLightVehicles
```
Comment (Mailaender) [inline mods/hv/chrome/ingame-player.yaml:562]: You need to add a hotkey.

Comment (Dzierzan) [reply mods/hv/chrome/ingame-player.yaml:562]: Yeah, I just noticed that.

Comment (Mailaender) [inline mods/hv/chrome/ingame-player.yaml:562]:
```suggestion
							Key: ProductionTypeWorkers
```

File: mods/hv/rules/bots.yaml [hunk 2]
```diff
@@ -22,8 +22,8 @@ Player:
 		ConstructionYardTypes: base
```
Comment (Mailaender) [inline mods/hv/rules/bots.yaml:22]:
```suggestion
		ConstructionYardTypes: base, base2
```

**Pull request merged**